source/default/_patterns/00-protons/tailwind.tokens.css

```diff
@@ -14,9 +14,81 @@ Example: Google Remote Fonts
 Serif - Google Fonts - Oswald
 Sans-Serif - Google Fonts - Open Sans
 Monospace - Google Fonts - Source Code Pro
-*/
+*/ 
 
 @import url(https://fonts.googleapis.com/css?family=OpenSans|Oswald|SourceCodePro&display=swap);
+@import url(https://fonts.googleapis.com/css2?family=Exo:wght@700&display=swap);
+
```

Comment:
Add Exo import here @bmartinez287 and @UTCbridget

Reply:
Works for me! That's what I had to do for the cards.